src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -149,12 +161,25 @@ public ExportConfiguration setExportDir(Path path) {
   }
 
   public ExportConfiguration setPemFile(Path path) {
-    this.pemFile = Optional.ofNullable(path);
+    //Check if path is a valid pem file with privateKey
+    if (PrivateKeyUtils.isValidPrivateKey(path)) {
+      this.pemFile = Optional.ofNullable(path);
+    }
     return this;
   }
 
-  public boolean isPemFilePresent() {
-    return pemFile.isPresent();
+  public ExportConfiguration removePemFile() {
```

Comment:
This is the only remove method. Other members rely on `Optional.ofNullable(value)` to sort that out. Please, mirror what we do on other members and remove this method unless it's really necessary.

Reply:
will remove this. Your next comment really shows its not  needed